koku/api/iam/test/tests_serializers.py

```diff
@@ -64,7 +68,7 @@ def test_create_user(self):
                 instance = serializer.save()
 
             self.assertEqual(user['username'], instance.username)
-            self.assertEqual(user['password'], instance.password)
+            self.assertIsNotNone(instance.password)
```

Comment:
Since I "fixed" the create for saving the object we don't get back the password we get back the hash, so unless I pulled the code to do the same hashing here there wasn't a simple way to perform and equality. Feel free to print out `instance.password` to get more clarity.

Reply:
`pbkdf2_sha256$100000$yI24Y7JgsZd3$8tjcNPG+7NZiN+ZSJne35p1S1kZ3ADHtrB22veIY51U=`